shared/actions/fs/index.tsx

```diff
@@ -988,6 +988,16 @@ const waitForKbfsDaemon = (state, action: ConfigGen.InstallerRanPayload | FsGen.
     })
 }
 
+const startManualCR = (state, action) =>
+  RPCTypes.SimpleFSSimpleFSClearConflictStateRpcPromise({
+    path: Constants.pathToRPCPath(action.payload.tlfPath),
+  }).then(() =>
+    FsGen.createTlfCrStatusChanged({
```

Comment:
I don't think this action should be triggered. Once ClearConflictState is called, the original conflicted TLF splits into two. Then we'll actually have two TLF entries in favorites list for the same TLF, one for the server view, and one for local branch. See [this](https://github.com/keybase/client/blob/master/protocol/avdl/keybase1/simple_fs.avdl#L373-L388). Looks like we should trigger favoritesLoad here.

Reply:
Yeah, you're right. Will fix.